torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -924,12 +924,19 @@ RegisterOperators reg2({
     DEFINE_INT_OP(aten::__lshift__, a << b),
     DEFINE_INT_OP(aten::__rshift__, a >> b),
 
-    DEFINE_GENERIC_BINARY_OP(aten::log, std::log(a) / std::log(b), float),
+    DEFINE_GENERIC_BINARY_OP(
+        aten::log,
+        std::log(a) / std::log(b),
+        float,
+        complex),
     DEFINE_INT_FLOAT_OP(aten::log, std::log(a) / std::log(b), float),
-    DEFINE_SCALAR_SCALAR_BINARY_OP(
+    DEFINE_INT_COMPLEX_OP(aten::log, std::log(a) / std::log(b), complex),
+    DEFINE_FLOAT_COMPLEX_OP(aten::log, std::log(a) / std::log(b), complex),
+    DEFINE_SCALAR_BINARY_OP_WITH_COMPLEX(
```

Comment:
```suggestion
    DEFINE_SCALAR_SCALAR_BINARY_OP_WITH_COMPLEX(
```
As discussed this should probably be `DEFINE_SCALAR_SCALAR_BINARY_OP_WITH_COMPLEX` to avoid naming conflicts with `aten::log` for tensors.